experimental/FTheoryTools/src/AbstractFTheoryModels/attributes.jl

```diff
@@ -1026,13 +1026,35 @@ Assuming that the first row of the given grading is the grading under Kbar
 
 Hypersurface model over a not fully specified base
 
-julia> zero_section(h)
+julia> zero_section_coordinates(h)
 3-element Vector{QQMPolyRingElem}:
  0
  1
  0
 ```
 """
+function zero_section_coordinates(m::AbstractFTheoryModel)
+  @req has_zero_section_coordinates(m) "No zero section coordinates stored for this model"
+  return get_attribute(m, :zero_section_coordinates)
+end
+
+
+@doc raw"""
+    zero_section(m::AbstractFTheoryModel)
+
+Return the zero section of a model as a cohomology class in the toric ambient space.
+If no zero section is known, an error is raised.
+This information is not typically available for
+Weierstrass and global Tate models, whose zero sections are known.
+
+```jldoctest
```

Comment:
```suggestion
```jldoctest; setup = :(Oscar.LazyArtifacts.ensure_artifact_installed("QSMDB", Oscar.LazyArtifacts.find_artifacts_toml(Oscar.oscardir)))
```

Reply:
I was wondering what was the purpose of this, if the tests seemingly passed. This changes makes sense.